src/phoenix/datasets/experiments.py

```diff
@@ -458,6 +458,7 @@ async def async_evaluate_run(
         max_retries=0,
         exit_on_error=False,
         fallback_return_value=None,
+        tqdm_bar_format=get_tqdm_progress_bar_formatter("running experiment evaluations"),
```

Comment:
@anticorrelator can this be the eval name?

Reply:
yeah this is higher level, we're grouping all the evals together and they're being done in parallel now

Reply:
hmmm, okay gotcha... I sorta want a progress bar per but can wait